web/src/routes/RegisterPage.tsx

```diff
@@ -1,5 +1,188 @@
 import React from 'react';
 
-export default function RegisterPage() {
-  return <div>RegisterPage</div>;
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+
+import BackgroundImage from '../assets/main-page.webp';
+import Header from '../components/Header';
+import { useTheme } from '../hooks/useTheme';
+
+// 핸드폰 번호 + 인증코드 제약 걸기
+// 이메일 regex
+
+const Container = styled.div`
+  width: 100vw;
+  height: 100vh;
+  background: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),
+    url(${BackgroundImage}) no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+`;
+
+const RegistContainer = styled.div`
+padding: 7vw;
+margin-left: calc(20vw - 5vmin);
+margin-right: calc(20vw - 5vmin);
+min-width : 30vh;
+height: 80vh;
+background-color: ${(props) => props.theme.colors.container};
+box-shadow: 0 10px 15px rgba(0, 0, 0, 0.3);
+`;
+
+const ContainerBox = styled.form`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+`;
+
+const Form = styled.form`
+  display: flex;
+  flex-direction: column;
+`;
+
+const Input = styled.input`
+  min-width : 5vh;
+  padding: 1.7vh;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  border: 3px solid #ebeff7;
+  border-radius: 16px;
+  font-size: 1.5vh;
+  font-weight: 600;
+  flex: 1;
+
+  &:focus {
+    outline: none;
+  }
+  
+  ::placeholder,
+  ::-webkit-input-placeholder {
+    opacity: 0.4;
+  }
+  :-ms-input-placeholder {
+    opacity: 0.4;
+  }
+`;
+
+const RegisterTitle = styled.span`
+  text-align: left;
+  font-size: 3vh;
+  font-weight: 1000;
+  margin-bottom : 2vh;
+  color: ${(props) => props.theme.colors.default};
+`;
+
+const FontText = styled.span`
+  margin-top: 5px;
+  text-align: left;
+  font-size: 1.3vh;
+  font-weight: 600;
+  color: ${(props) => props.theme.colors.default};
+`;
+
+const PhoneNumberBox = styled.button`
+  border: 3px solid #ebeff7;
```

Comment:
We have a global `border` value here: https://github.com/kookmin-sw/capstone-2023-40/blob/feature/login-regist/web/src/types/styled.d.ts#L8 

Please make use of reusable values not hard coded one.